### ISSUE TITLE
Save Most Recent Message - documentation bug

### ISSUE THREAD

**Issue by michalk-k** [drmibell/node-red-contrib-queue-gate#18]
in section "Save Most Recent Message" of node documentation, the example is missing "Keep Newest Messages" feature enabled. It results in returning oldest message instead of most recent.

with regards

**Comment by drmibell** [comment]
Good catch, thanks. I assume you have fixed this in your copy of the flow and are ok with the results. I have been working on a bug-fix update of the node, so it will be a few days before the GitHub and npm versions are corrected.

**Comment by michalk-k** [comment]
Yeah, since it was the major feature I wanted to use your node for, it was pretty easy to spot and set properly. Thanks for asking.
